src/lib/bootloader/stage1_proposal.rb

```diff
@@ -198,7 +199,9 @@ class PPC < Stage1Proposal
       def propose
         partition = proposed_prep_partition
         if partition
-          assign_bootloader_device([:custom, partition])
+          # ensure that stage1 device is in udev (bsc#1041692)
```

Comment:
Well, it is about priorities. I spend almost whole day implementing and testing it. Of course with my English skill, I can easily spend another couple of hours to try to produce precise an English sentences, but I have other tasks in my queue and in scrum board.
So for me question is where my time bring the most value to customers and also to rest of team. So if a comment in a code is understandable and developer understand what I want to say, even if not precisely and perfectly correct, it is enough for me.